scale-tests/scripts/run.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+set -ex
+SCRIPT_DIR="$(realpath "$(dirname "$0")")"
+SCALE_TESTS_DIR="$(dirname "${SCRIPT_DIR}")"
+PROJECT_ROOT_DIR="$(dirname "${SCALE_TESTS_DIR}")"
+
+TEMPLATES_DIR="${SCALE_TESTS_DIR}/templates"
+
+NAMESPACE_PREFIX=${NAMESPACE_PREFIX:-spark}
+APP_NAME_PREFIX=${APP_NAME_PREFIX:-spark-sort}
+
+S3_PATH=${S3_PATH:-spark-sort}
+NUM_EXECUTORS=${NUM_EXECUTORS:-4}
+
+if [[ $# -lt 2 ]]; then
+  echo "Usage:" >&2
+  echo "  $0 <number of applications to create> <number of namespaces to submit apps to>" >&2
+  exit 1
+fi
+
+. ${SCRIPT_DIR}/aws_credentials.sh
+
+for n in $(seq ${2}); do
+#    NAMESPACE="${NAMESPACE_PREFIX}-${n}"
+    NAMESPACE=spark
```

Comment:
This line looks like a leftover from local testing

Reply:
fixed